.devcontainer/docker-compose.yml

```
@@ -0,0 +1,67 @@
+      test: [ "CMD-SHELL", "pg_isready -U dozens" ]
+      interval: 5m
+      timeout: 2m
+      retries: 5
+      start_period: 2m
+      start_interval: 2m
+volumes:
+  primary_hhd_dozens:
+
+networks:
+  dozens-ecosystem:
+    external: true
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)